crates/autopilot/src/arguments.rs

```diff
@@ -167,6 +167,14 @@ pub struct Arguments {
     /// limit orders.
     #[clap(long, env, use_value_delimiter = true)]
     pub quoting_strategies: Vec<QuotingStrategy>,
+
+    /// Enable the colocation run loop.
+    #[clap(long, env)]
+    pub enable_colocation: bool,
+
+    /// Driver base URLs for colocation.
```

Comment:
```suggestion
    /// Driver base URLs.
```

"The colocation" will just be the default state of affairs, so I don't think this is needed. We should more or less just forget the term as soon as we're done with it.